src/store/tests/test-data/cat-manoeuvres/manoeuvres/manoeuvres.cat-cm.reducer.ts

```diff
@@ -0,0 +1,79 @@
+import { createReducer, on } from '@ngrx/store';
+import { CompetencyOutcome } from 'src/app/shared/models/competency-outcome';
+import { CatCMUniqueTypes } from '@dvsa/mes-test-schema/categories/CM';
+import { CatCEMUniqueTypes } from '@dvsa/mes-test-schema/categories/CEM';
+import { CatC1MUniqueTypes } from '@dvsa/mes-test-schema/categories/C1M';
+import { CatC1EMUniqueTypes } from '@dvsa/mes-test-schema/categories/C1EM';
+import * as manoeuvresActions from '../../common/manoeuvres/manoeuvres.actions';
+
+export const initialState: CatCMUniqueTypes.Manoeuvres |
+CatCEMUniqueTypes.Manoeuvres |
+CatC1MUniqueTypes.Manoeuvres |
+CatC1EMUniqueTypes.Manoeuvres = {
+  reverseManoeuvre: {},
+};
+
+export const manoeuvresCatManoeuvreReducer = createReducer(
+  initialState,
+  on(
+    manoeuvresActions.RecordManoeuvresSelection,
+    (state, { manoeuvre }) => ({
+      [manoeuvre]: {
+        ...state[manoeuvre],
+        selected: true,
+      },
+    }),
+  ),
+  on(
```

Comment:
There are no driving faults on manoeuvre categories, so we should be able to remove this